tests/functional/spec/interstitial.spec.js

```diff
@@ -17,6 +17,7 @@ describe('Interstitial Pages', function () {
 
         // When
         expect(InterstitialPage.getMainHeading()).to.contain('Breakfast Interstitial Page')
```

Comment:
We could have another one, given I am answering a questionnaire with an interstitial page, when I am on the interstitial page, then the button should say continue